contracts/cw-names-registry/src/error.rs

```diff
@@ -0,0 +1,44 @@
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ContractError {
+    #[error("{0}")]
+    Std(#[from] StdError),
+
+    #[error("Unauthorized")]
+    Unauthorized {},
+
+    #[error("Insufficient funds sent")]
```

Comment:
See note above - something like `"Wrong payment amount for name registration sent."`

Error might be `IncorrectPaymountAmount`.